### PR TITLE
Query Metadata Docs

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: tools/public_api_guard/core/index.d.ts [hunk 1]
```diff
@@ -265,8 +265,30 @@ export declare abstract class ComponentRef<C> {
...
+        read?: any;
+    }): ContentChild;
+}
+
+/** @stable */
 export declare const ContentChildren: ContentChildrenDecorator;
 
+/** @stable */
+export interface ContentChildrenDecorator {
+    /** @stable */ (selector: Type<any> | Function | string, {descendants, read}?: {
```
Comment (vicb) [inline tools/public_api_guard/core/index.d.ts]: I don't think `/** @stable */` is right here ?

/cc @tbosch

Comment (vicb) [inline tools/public_api_guard/core/index.d.ts]: Wait what ?

A doc change that update a .d.ts file ?

Comment (vsavkin) [inline tools/public_api_guard/core/index.d.ts]: Decorators are special. We don't document the symbol itself, but rather it's "call" function.

File: modules/@angular/examples/core/di/ts/contentChild/content_child_howto.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+@Directive({selector: 'child-directive'})
+class ChildDirective {
+}
+
+@Directive({selector: 'someDir'})
+class SomeDir {
+  @ContentChild(ChildDirective) contentChild;
+
+  ngAfterContentInit() {
```
Comment (IgorMinar) [inline modules/@angular/examples/core/di/ts/contentChild/content_child_howto.ts]: add interface to the class?

**Review by vicb — CHANGES_REQUESTED** [review]
- move API changes outside of docs commit,
- fix the CI

**Comment by alexeagle** [comment]
I merged this PR prematurely onto master. Then I reverted the commits in 5fad37d

Sorry for the noise, carry on...

**Comment by IgorMinar** [comment]
2c42a50..003294d

**Pull request closed without merging**